daffodil-japi/src/main/scala/org/apache/daffodil/japi/infoset/Infoset.scala

```diff
@@ -123,85 +123,85 @@ abstract class InfosetOutputter extends SInfosetOutputter {
   /**
    * Called by Daffodil internals to signify the beginning of the infoset.
    *
-   * @return true on sucess, false if there was an error and Daffodil should stop all
-   *         future calls to the InfosetOutputter
+   * @throws Exception if there was an error and Daffodil should stop parsing
    */
-  def startDocument(): Boolean
+  @throws[Exception]
```

Comment:
I'm not sure why these javadoc errors are happening (java.lang.Exception should be built-in), but DAFFODIL-2721 says we may want to create specific exception(s) that both the inputter and outputter should be expected to throw when errors happen.  I think if you create such exception(s) (their names could be InfosetException or InfosetInputterException/InfosetOutputterException, take your choice), the javadoc errors would go away.  I also wonder if the specific exception should be a checked exception or inherit from RuntimeException (less code would be needed in the latter case).

Reply:
Yeah, I can't figure it out. Seems to be a known issue with scaladoc--I've found lot's of other people with similar issues and no solution. Though it's not clear to me why it works for some exceptions types but not others. For now, I've just removed the `@throws` in the comments.

This will probably go away once we add a custom exception. My initial goal here is to just fix the sax/exi regressions as quickly as possible since I know of active development/testing around that.